packages/victory-voronoi/src/helper-methods.js

```diff
@@ -50,6 +54,18 @@ const getCalculatedValues = (props) => {
 
   let data = Data.getData(props);
   data = Data.formatDataFromDomain(data, domain);
+  // Manually remove data with null _x or _y values.
+  // Otherwise, we hit null error in: d3-voronoi/src/Cell.js
+  data = data.filter((datum) => {
```

Comment:
@becca-bailey Can you review this change? I've added a direct check + filter for the thing d3-voronoi blows up on, and it passes tests, but want to have some substantive eyes on it :)

Reply:
If it passes tests, this works for me! If we come back and do a more substantial refactor it might be worth evaluating whether we want to move this filter into the `formatDataFromDomain` function and apply it everywhere, but I don't think it's necessary to figure that out in this PR.